db/silkworm/stagedsync/util.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+   Copyright 2020-2021 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+#include <silkworm/common/base.hpp>
+
+#ifndef SILKWORM_STAGEDSYNC_UTIL_HPP_
+#define SILKWORM_STAGEDSYNC_UTIL_HPP_
+
+/*
+Part of the compatibility layer with the Erigon DB format;
+see its package dbutils.
+*/
+
+namespace silkworm::stagedsync {
+
+enum class [[nodiscard]] StageResult {
+    kStageSuccess,
+    kStageBadChainSequence,
+    kStageInvalidRange,
+    kStageAborted
+};
+
+void check_stagedsync_error(StageResult code);
+
+ /*
+ * Extract the incarnation from an encoded account object without fully decoding it.
+ */
+uint64_t extract_incarnation(ByteView);
+
+// Return db formatted (key, value) pair
+std::pair<Bytes, Bytes> convert_to_db_format(Bytes& key, Bytes& value);
```

Comment:
What does this method do? The name and the comment are too generic, I struggle to understand what it's for.
Also, please take the arguments as `ByteView`  or `const Bytes&` unless you're changing them. Rationale: [Google Style Guide](https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs) and https://rules.sonarsource.com/cpp/RSPEC-6009

Reply:
I put a comment that makes more sense now. it is the FromDbFormat function from erigon.